StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet.swift

```diff
@@ -286,6 +286,22 @@ extension PaymentSheet: PaymentSheetViewControllerDelegate {
             self.completion?(result)
         }
     }
+    func paymentSheetViewControllerFinishedOnPay(_ paymentSheetViewController: PaymentSheetViewController,
+                                                 completion: (() -> Void)? = nil) {
+        self.bottomSheetViewController.transitionSpinnerToComplete(animated: true) {
+            self.bottomSheetViewController.removeBlurEffect(animated: true, completion: completion)
+        }
```

Comment:
If we don't care about the animation of the button turning from blue to green, we can make this:
```
-            self.bottomSheetViewController.removeBlurEffect(animated: true, completion: completion)
+            if let completion {
+                completion()
+            }
+            self.bottomSheetViewController.removeBlurEffect(animated: true)
```

Reply:
Design is in flux, we can always change it later.